src/App.tsx

```diff
@@ -1,26 +1,10 @@
 import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+interface AppProps {}
 
-function App() {
+export const App: React.FC<AppProps> = ({}) => {
```

Comment:
It seems like we want this component to be rather lightweight. Do you think we'll have to refactor the AppProps interface at some point? Good call with the named export!